src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs

```diff
@@ -15,6 +15,7 @@
 // </copyright>
 
 using System.Diagnostics;
+using System.Diagnostics.Metrics;
```

Comment:
Is this using statement necessary? I'm not seeing any references to anything in it.

Reply:
To support `cref` in comment we need `System.Diagnostics.Metrics` namespace. `System.Diagnostics` namespace in this file is also used for the same reason.

```
       /// <summary>
        /// Configuration key for additional <see cref="Meter"/> names to be added to the meter at the startup.
        /// </summary>
        public const string AdditionalSources = "OTEL_DOTNET_AUTO_METRICS_ADDITIONAL_SOURCES";
```